src/test/java/dev/vality/analytics/utils/RateSinkEventTestUtils.java

```diff
@@ -17,6 +18,28 @@
 import static io.github.benas.randombeans.api.EnhancedRandom.randomListOf;
 
 public class RateSinkEventTestUtils {
+    public static void main(String[] args) {
+        try {
+            List<Quote> quotes = randomListOf(4, Quote.class);
+            Change created = Change.created(
+                    new ExchangeRateCreated(
+                            new ExchangeRateData(
+                                    new TimestampInterval(
+                                            Instant.now().toString(),
+                                            Instant.now().toString()
+                                    ),
+                                    quotes
+                            )
+                    )
+            );
+            TFieldIdEnum[] tFieldIdEnums = created.getFields();
+            System.out.println(tFieldIdEnums);
+            Geck.toMsgPack(created);
```

Comment:
java.lang.AbstractMethodError: Receiver class dev.vality.xrates.rate.Change does not define or inherit an implementation of the resolved method 'abstract org.apache.thrift.TFieldIdEnum[] getFields()' of interface org.apache.thrift.TBase.

![image](https://user-images.githubusercontent.com/19729841/152019020-91d5530e-be41-44ea-9270-f2adea31de44.png)

Reply:
https://github.com/valitydev/analytics/runs/5024219214?check_suite_focus=true#step:3:3879